handler.go

```diff
@@ -54,6 +54,9 @@ type packageHandler struct {
 
 func (h packageHandler) Handle(w http.ResponseWriter, r *http.Request, ps httprouter.Params) {
 	canonicalURL := fmt.Sprintf("%s/%s", h.config.URL, h.pkgName)
+	if h.pkg.URL != "" {
+		canonicalURL = fmt.Sprintf("%s/%s", h.pkg.URL, h.pkgName)
+	}
```

Comment:
minor information flow nit:
I think I'd like for us to construct the component of the URL first and then build the canonical URL. This versus building the URL, and building it again based on condition. This way, if we add customization of other components (however unlikely), that would also happen right above `canonicalURL := ...`.

```suggestion
	baseURL := h.config.URL
	if h.pkg.URL != "" {
		baseURL = h.pkg.URL
	}
	canonicalURL := fmt.Sprintf("%s/%s", baseURL, h.pkgName)
```